nixos/modules/services/x11/desktop-managers/plasma5.nix

```
@@ -344,6 +344,7 @@ in
 
           qtvirtualkeyboard
 
+          qt5.qttools
```

Why do we want this?

---

https://github.com/NixOS/nixpkgs/issues/173544

---

That is for now fixed by wrapping. see https://github.com/NixOS/nixpkgs/search?q=qdbus

---

So, I should remove it?

---

Its only used by some programs so they should probably wrap it.

---

The problem of wrapping it is that, in the case of `joystickwake`, that `qdbus` is needed only on Plasma Wayland. It's not necessary on Gnome, for instance, so the combo Gnome+joystickwake will install an unneeded dependency, and it's impossible to install this dependency only if the user is using Plasma (unless `joystickwake` is turned into an option). There are probably other script that assume "The user is using Plasma => `qdbus` is in PATH". One of them is... [the "useful commands" collection from the official KDE userbase wiki](https://userbase.kde.org/KDE_Connect/Tutorials/Useful_commands) (9 matches).
Finally, `qttols` is already installed with Plasma (it's for example a dependency of `spectacle`). With this, it's just exposed in the PATH (maybe I should add a comment for that).

Still, you have the final say.

---

I think that changes things a bit.

but I think we want to do the following

```suggestion
    (lib.getBin qttools)
```